docs/database/database-spans.md

```diff
@@ -60,15 +60,12 @@ Database spans MUST follow the overall [guidelines for span names](https://githu
 
 The **span name** SHOULD be `{db.query.summary}` if a summary is available.
 
-<!-- markdown-link-check-disable -->
-<!-- HTML anchors are not supported https://github.com/tcort/markdown-link-check/issues/225-->
```

Comment:
@chalin just wanted to make sure these aren't needed for website? thanks!

Reply:
Thanks for the ping. All good from the OTel website side of things.